modules/product/src/Entity/ProductVariation.php

```diff
@@ -171,6 +171,24 @@ public function setOwnerId($uid) {
   /**
    * {@inheritdoc}
    */
+  public function getLineItemType() {
+    // Get the product variation type value from the entity.
+    $variationTypeValue = $this->getType();
+
+    // Get the variationType entity based on the variation type value.
+    $variationType = $this->entityManager()
+      ->getStorage('commerce_product_variation_type')
+      ->load($variationTypeValue);
+
+    // Get the lineItemType value from the variation type entity.
+    $lineItemType = $variationType->get('lineItemType');
```

Comment:
Always use a getter when you can. So in this case $variationType->getLineItemType();